rhizome/host/lib/spdk_setup.rb

```diff
@@ -85,6 +85,10 @@ def install_package
     end
   end
 
+  def hugepage_count
+    2
+  end
```

Comment:
This seems like a lot of indirection to insert "2" somewhere, do you have a follow-up intended that makes this dynamic in some interesting way?

Reply:
This will most likely become dynamic in future as max number of volumes per host will become non-constant. However, thinking more about it, I think we don't know what the correct change for those cases before we have an example. So, I removed the indirection to keep this change as simple as possible & defer that to future.